symmer/symplectic/noncontextual_op.py

```diff
@@ -137,6 +145,21 @@ def _single_sweep_noncontextual_operator(cls, H, strategy='magnitude'):
         
         return cls.from_PauliwordOp(noncontextual_operator)
 
+    @classmethod
+    def _from_basis_noncontextual_op(cls, H: PauliwordOp, basis: PauliwordOp):
```

Comment:
For use with the from_hamiltonian initialization method for NoncontextualOp - the users specifies a noncontextual generating set and a corresponding noncontextual Hamiltonian is built by taking any terms in H that may be constructed under the Jordan product.

Reply:
Nice!

Reply:
(we should include the Jordan product definition in our doc strings)